vendor/wp-now/src/execute-wp-cli.ts

```diff
@@ -49,6 +46,11 @@ export async function executeWPCli ( projectPath: string, args: string[] ): Prom
 		// @see https://github.com/wp-cli/wp-cli/issues/1102
 		putenv( 'SHELL_PIPE=0' );
 
+		// When running PHP on Playground, the value of constant PHP_OS is set to Linux.
+		// This implies that platform-specific logic won't work as expected. To solve this,
+		// we use an environment variable to ensure WP-CLI runs the Windows-specific logic.
+		putenv( 'WP_CLI_TEST_IS_WINDOWS=${isWindows ? 1 : 0}' );
```

Comment:
This environment variable is mostly used by unit tests in `wp-cli`. As described in the inline comment, it allows us to tell `wp-cli` when the command is being executed in Windows. It should be enough as a temporary workaround but in the future, we should plan a more robust solution to match the value provided in `PHP_OS` with the actual OS.